controllers/horizontal_runner_autoscaler_webhook.go

```diff
@@ -246,7 +246,7 @@ func (autoscaler *HorizontalRunnerAutoscalerGitHubWebhook) Handle(w http.Respons
 			if target != nil {
 				if e.GetAction() == "queued" {
 					target.Amount = 1
-				} else if e.GetAction() == "completed" {
+				} else if e.GetAction() == "completed" && e.GetConclusion() != "skipped" {
```

Comment:
```suggestion
				} else if e.GetAction() == "completed" && e.GetWorkflowJob().GetConclusion() != "skipped" {
```

Apparently we needed to drill it down to actually get conclusion